pom.xml

```diff
@@ -149,34 +142,33 @@
     </build>
     <dependencies>
         <dependency>
+            <groupId>io.logz.sender</groupId>
+            <artifactId>logzio-sender</artifactId>
+            <version>1.0.1</version>
+        </dependency>
+        <dependency>
             <groupId>ch.qos.logback</groupId>
             <artifactId>logback-classic</artifactId>
             <version>1.1.7</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>com.bluejeans.common</groupId>
-            <artifactId>bigqueue</artifactId>
-            <version>1.7.0.4</version>
-        </dependency>
-        <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>19.0</version>
-        </dependency>
-        <dependency>
-            <groupId>com.google.code.gson</groupId>
-            <artifactId>gson</artifactId>
-            <version>2.6.2</version>
+            <version>20.0</version>
         </dependency>
-
         <dependency>
             <groupId>org.assertj</groupId>
             <artifactId>assertj-core</artifactId>
             <version>3.4.1</version>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>io.logz.sender</groupId>
+            <artifactId>logzio-sender-test</artifactId>
+            <version>1.0.1</version>
```

Comment:
Use properties and have both logzio-sender-test and logzio-sender use the same version